library/oneview_appliance_device_read_community.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: oneview_appliance_device_read_community
+short_description: Manage the Appliance Device Read Community string.
+description:
+    - Provides an interface to manage the Appliance Device Read Community string. It can only update it.
+      This results in an update of the community string on all servers being managed/monitored by this OneView instance.
+      The supported characters for community string are aA-zA, 0-9, !, ", #, $, %, ', &, (, ), *, +, -, ., /, @, `, {, |, }.
+version_added: "2.7"
```

Comment:
this cannot be more than 2.5

Reply:
Ok..Fixed in commit 7c64b78230b12ccb0d93b1ac0d847f68a538e27f